crates/wallet/src/wallet/error.rs

```diff
@@ -228,6 +228,14 @@ pub enum BuildFeeBumpError {
     IrreplaceableTransaction(Txid),
     /// Node doesn't have data to estimate a fee rate
     FeeRateUnavailable,
+    /// Happens when the descriptor is impossible to satisfy
+    MiniscriptError(miniscript::Error),
```

Comment:
Can we write a test to show that it's possible to hit this error?

Reply:
Sure, I'll try working on that.

Reply:
I'll be working on that in another issue as you suggested, #1485